docs/fundamentals/runtime-libraries/snippets/System/Span/Overview/csharp/program.cs

```diff
@@ -20,18 +20,19 @@ private static void CreateSpanFromArray()
     {
         // <Snippet1>
         // Create a span over an array.
-        var array = new byte[100];
-        var arraySpan = new Span<byte>(array);
+        byte[] array = new byte[100];
```

Comment:
You could replace these two lines with this (unless this sample is supposed to show conversion from array to Span):

```suggestion
        byte[] array = new byte[100];
```